### PR TITLE
feat(payment): PI-1366 e2e tests for Adyen integration on Checkout JS

### PR DESCRIPTION
**Pull request by bc-ania**
## What?
Added E2E tests for Adyen integration on Checkout JS

## Why?
https://bigcommercecloud.atlassian.net/browse/PI-1366

## Testing / Proof
All tests have passed 

@bigcommerce/team-checkout

### CODE REVIEW

**Review by vitalii-koshovyi — APPROVED** [review]
LGTM!

File: packages/adyen-integration/jest.config.js [hunk 1]
```diff
@@ -9,4 +9,5 @@ module.exports = {
     },
     setupFilesAfterEnv: ['../../jest-setup.ts'],
     coverageDirectory: '../../coverage/packages/adyen-integration',
+    testPathIgnorePatterns: ['/adyenv2/e2e/', '/adyenv3/e2e/'],
```
Comment (bc-peng) [inline packages/adyen-integration/jest.config.js:12]: Do we really need this here? Thought we ignored the `/e2e` on a global level.
https://github.com/bigcommerce/checkout-js/blob/64814f14bcf585f8f52345c9b64cddf1334fa3f9/jest.preset.js#L6

Comment (bc-ania) [reply packages/adyen-integration/jest.config.js:12]: without it, adyen e2e tests are running with unit tests

Comment (bc-peng) [reply packages/adyen-integration/jest.config.js:12]: I see. Could you please move e2e tests to `packages/adyen-integration/e2e` as we put all e2e tests in this folder  of each package?

**Pull request merged**